client/app/_components/layout/SettingDropdown/SettingDropdown.tsx

```diff
@@ -20,6 +20,7 @@ const SettingDropdown = () => {
       onSuccess: () => {
         resetAllStores()
         queryClient.clear()
+        queryClient.invalidateQueries({ queryKey: ['me'] })
```

Comment:
Can I ask you why you need this invalidateQueries after clearing all the cache?

Reply:
Because this queryClient.clear() doesn't clear if you're using cached data at the moment. There is a possibility that GetMe data is still using in some components at this moment, so I needed to add this (if I remember correctly)